extensions/commands/sbom/cmd_cyclonedx.py

```diff
@@ -58,35 +61,25 @@ def cyclonedx(conan_api: ConanAPI, parser, *args) -> 'Bom':
         # if loading dependencies is performed outside the actual conan-command in global/module scope.
         print('The sbom extension needs an additional package, please run:',
               # keep in synk with the instructions in `README.md`
-              "pip install 'cyclonedx-python-lib>=3.1.5,<5.0.0'",
+              "pip install 'cyclonedx-python-lib>=5.0.0,<6",
               sep='\n', file=sys.stderr)
         sys.exit(1)
 
     if TYPE_CHECKING:
         from conans.client.graph.graph import Node
 
-    def cyclonedx_major_version_is_4() -> bool:
-        try:
-            LicenseChoice(license=LicenseFactory().make_from_string("MIT"))
-            return True
-        except TypeError:  # argument in version 3 is named license_
-            return False
-
     def package_type_to_component_type(pt: str) -> ComponentType:
         return ComponentType.APPLICATION if pt == "application" else ComponentType.LIBRARY
 
-    def licenses(ls: Optional[Union[Tuple[str, ...], Set[str], List[str], str]]) -> Optional[Iterable[LicenseChoice]]:
+    def licenses(ls: Optional[Union[Tuple[str, ...], Set[str], List[str], str]]):
         """
         see https://cyclonedx.org/docs/1.4/json/#components_items_licenses
         """
         if ls is None:
             return None
         if not isinstance(ls, (tuple, set, list)):
             ls = [ls]
-        if cyclonedx_major_version_is_4():
-            return [LicenseChoice(license=LicenseFactory().make_from_string(i)) for i in ls]  # noqa
-        else:
-            return [LicenseChoice(license_=LicenseFactory().make_from_string(i)) for i in ls]
+        return [LicenseFactory().make_from_string(i) for i in ls]  # noqa
```

Comment:
why is that `# noqa` needed?

Reply:
sorry, they were required when parts of the code were not tested in the CI, I dropped it